vignettes/rtestim.Rmd

```diff
@@ -0,0 +1,179 @@
+$\lambda$s for which the $\Rtseq$ is calculated
+
+
+## Cross Validation
+
+`{rtestim}` also provides a cross validation procedure for selecting the amount of smoothness to be used in the final estimate (leave-every-k-th-out cross validation). Minimizing this metric, in principle, balances prediction error
+and smoothness (`lambda.min`) though if smoother estimates are desired, one
+can instead use `lambda.1se`, the largest value of $\lambda$ within one standard
+error of the minimum.
+
+```{r}
+mod_cv <- cv_estimate_rt(observed_counts = case_counts, nsol = 20)
+```
+
+The following command plots the cross validation errors for each $\lambda$ in
+ascending order.
+
+```{r}
+plot(mod_cv)
+```
+
+The plot above displays vertical lines that correspond to the cross-validation scores 
+for specific values of $\lambda$. The blue point at the center of each line 
+represents the mean score for that value of $\lambda$ across all cross-validation
+folds. The top and bottom caps of each line indicate one cross-validation 
+standard error above and below the mean score for the given value of $\lambda$ 
+across all cross-validation folds. Two special values of $\lambda$'s are 
+highlighted with dashed lines. The one on the left represents the $\lambda$ that
+gives minimum mean cross-validated error, called `lambda.min`, and the one on the
+right gives the most regularized model such that the cross-validated error is 
+within one standard error of the minimum, called `lambda.1se`.
+
+Users may wish to visualize the particular $\Rtseq$ which minimizes the 
+cross-validation error while prioritizing smoothness.
+
+```{r}
+plot(mod_cv, which_lambda = 'lambda.1se')
+```
+
+
+## Uneven Reporting Frequency
+
+Ideally, case counts are observed at regular intervals, such as daily or 
+weekly, but this is not always the case. 
+`{rtestim}` also accommodates scenarios in which cases are reported 
+with uneven intervals. To demonstrate this, we generate a sequence of integers
+representing the days at which we observe the case counts
+
+```{r}
+observation_incr <- rpois(101, lambda = 1)
+observation_incr[observation_incr == 0] = 1
+observation_time <- cumsum(observation_incr)
+```
+
+We can then fit the model by passing the observation time point as `x`
+
+```{r}
+mod <- estimate_rt(observed_counts = case_counts, x = observation_time)
+plot(mod)
+```
+
+
+## Changing degree of difference operator
+
+The degree of the difference operator $k$ defaults to 3 for the algorithm, which
+corresponds to a piecewise cubic estimate $\Rtseq$. To estimate
+$\Rtseq$ with piecewise constant curves for example, use the command
+
+```{r}
+mod <- estimate_rt(observed_counts = case_counts, degree = 1)
+plot(mod)
+```
+
+
+
+# Example - Canadian Covid-19 cases
+
+Finally, we use a long history of real case counts in Canada. The data is
+available from [opencovid.ca](https://opencovid.ca) and a version is included
```

Comment:
It might be clearer to include the date of data availability, say "the version of data collected on July 4 is included".